drivers/network/afd/afd/tdi.c

```diff
@@ -229,13 +231,16 @@ NTSTATUS TdiQueryMaxDatagramLength(
     PTDI_MAX_DATAGRAM_INFO Buffer;
     NTSTATUS Status = STATUS_SUCCESS;
 
-    Buffer = ExAllocatePool(NonPagedPool, sizeof(TDI_MAX_DATAGRAM_INFO));
+    Buffer = ExAllocatePoolWithTag(NonPagedPool,
```

Comment:
I guess this one could also have its own tag. But I guess you have enough tags, so fair enough :p

Reply:
We do indeed have so many tags, but we're not running out of them anytime soon are we? :)
If you would like to see this have its own tag I'm all for it.

Reply:
Nah, this seems fine for an initial run through. If someone finds more leaks later maybe they'll refine it more.